### PR TITLE
My VA: removes EVSS claims code

### PR DESCRIPTION
**Pull request by dellerbie**
Removes EVSS claims code. We are 100% using Lighthouse `v0/benefits_claims` endpoint on My VA.

## Summary

- Removes usage of the feature toggle `myVaUseLighthouseClaims`
- Removes EVSS claims code from the My VA application
- Updates specs

## Related issue(s)

- [59817](https://github.com/department-of-veterans-affairs/va.gov-team/issues/59817)

## Testing done

- Updated and ran specs
- Visual confirmation locally
- More manual testing on staging to be performed

## What areas of the site does it impact?
My VA

## Acceptance criteria
- [ ] Specs are passing
- [ ] No regressions introduced to My VA

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### CODE REVIEW

File: src/applications/personalization/dashboard/components/claims-and-appeals/ClaimsAndAppeals.jsx [hunk 1]
```diff
@@ -266,7 +248,6 @@ const mapStateToProps = state => {
 const mapDispatchToProps = {
   loadAppeals: getAppealsAction,
   loadClaims: getClaimsAction,
-  loadLighthouseClaims: getLighthouseClaimsAction,
```
Comment (adamwhitlock1) [inline src/applications/personalization/dashboard/components/claims-and-appeals/ClaimsAndAppeals.jsx:269]: minor comment here, I try and keep the naming the same with mapDispatchToProps across the imported actions, so when importing the action you rename `getClaims` to `getClaimsAction`, then in mapDispatchToProps you can just use the original function name:

`getClaims: getClaimsAction`

That way there aren't two different naming conventions of load vs get and it makes things a little easier to follow. Not blocking whatsoever though.

File: src/applications/personalization/dashboard/tests/components/claims-and-appeals/ClaimsAndAppealsLighthouse.unit.spec.jsx [hunk 2]
```diff
@@ -151,13 +148,10 @@ describe('ClaimsAndAppeals component', () => {
...
-          <ClaimsAndAppeals dataLoadingDisabled useLighthouseClaims />,
-          {
-            initialState,
-            reducers,
-          },
-        );
+        view = renderInReduxProvider(<ClaimsAndAppeals dataLoadingDisabled />, {
+          initialState,
+          reducers,
+        });
```
Comment (adamwhitlock1) [inline src/applications/personalization/dashboard/tests/components/claims-and-appeals/ClaimsAndAppealsLighthouse.unit.spec.jsx:154]: As a side note, this `dataLoadingDisabled` feels like a code smell to me. Maybe in the future we can look at what we could do to not have this required for testing.

File: src/applications/personalization/dashboard/tests/components/claims-and-appeals/ClaimsAndAppeals.unit.spec.jsx [hunk 3]
```diff
@@ -62,28 +62,24 @@ function makeAppealObject({ updateDate, closed = false }) {
...
-      developmentLetterSent: false,
-      decisionLetterSent: false,
-      phase,
-      everPhaseBack: false,
-      currentPhaseBack: false,
-      requestedDecision: false,
+      claimDate: dateFiled || '2021-01-21',
+      claimPhaseDates: {
+        phaseChangeDate: updateDate,
+      },
```
[the remark this answers is not in the document]
Comment (jerekshoe) [reply src/applications/personalization/dashboard/tests/components/claims-and-appeals/ClaimsAndAppeals.unit.spec.jsx:73]: Yeah don’t ask me how they came up with their naming scheme, but these are the correct field names

**Review by adamwhitlock1 — DISMISSED** [review]
This is a beast! Thanks Derrick for getting through this massive cleanup effort.

Had a couple minor things that I already commented on.

There were a few warnings about axe checks not being done in cypress e2e tests. For those you should be able to sprinkle in the cypress command `cy.injectAxeThenAxeCheck();` to get those cleared up.

File: src/applications/personalization/dashboard/tests/components/claims-and-appeals/ClaimsAndAppealsLighthouse.unit.spec.jsx
Comment (allisonlu) [inline src/applications/personalization/dashboard/tests/components/claims-and-appeals/ClaimsAndAppealsLighthouse.unit.spec.jsx:1]: Would these tests replace the ones in `ClaimsAndAppeals.unit.spec.jsx`? Do we have to keep both files?

**Review by allisonlu — DISMISSED** [review]
^ what Adam said! This looks great

**Comment by DanielMiller-agile6** [comment]
I updated this branch and got tests passing again.  There were a few conflicts that I had to resolve with more recent changes in main, so this could use a fresh set of reviews :)

File: src/applications/personalization/dashboard/components/claims-and-appeals/ClaimsAndAppeals.jsx (hunk 1 above)
Comment (DanielMiller-agile6) [reply src/applications/personalization/dashboard/components/claims-and-appeals/ClaimsAndAppeals.jsx:269]: resolved: renamed `loadAppeals` and `loadClaims` to `getAppeals` and `getClaims` in this file for consistency.

**Review by allisonlu — APPROVED** [review]
lgtm! great work!!

**Review by adamwhitlock1 — APPROVED** [review]
this all looks pretty good to me. Thanks for cleaning this up and getting conflicts resolved. its a big PR

**Comment by DanielMiller-agile6** [comment]
related vets-api PR:
https://github.com/department-of-veterans-affairs/vets-api/pull/14849

**Review by harshilmoradia — APPROVED** [review]

**Pull request merged**